src/mono/netcore/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.xml

```diff
@@ -692,10 +692,5 @@
 			<!-- marshal-ilgen.c:emit_invoke_call -->
 			<method signature="System.Void .ctor()" />
 		</type>
-
-		<!-- Used by binary formatter tests -->
-		<type fullname="System.Threading.ThreadStartException">
```

Comment:
The PR isn't changing any tests.  Does that mean the comment is stale?

Reply:
Possibly but at the same time we don't run any tests with trimmed libraries

Reply:
The libraries that the tests run against are per-assembly trimmed. And the tests are still failing when this is not pinned as the CI shows.